tests/HUBViewControllerTests.m

```diff
@@ -2536,4 +2559,9 @@ - (void)registerAndGenerateComponentsWithNamespace:(NSString *)namespace
     };
 }
 
+- (void)assertEdgeInsetsEqualInsets:(UIEdgeInsets)insets
```

Comment:
Taken out of context this method name looks a bit off. I would suggest either just using a local block to perform these calculations, or naming it a bit more descriptive (like `assertCollectionViewContentInsetEqualToEdgeInsets`). What do you think?